docs/endpoints/post-identity-map.md

```diff
@@ -68,26 +70,30 @@ The following are unencrypted JSON request body examples for each parameter, one
     "email_hash":[
         "tMmiiTI7IaAcPpQPFQ65uMVCWH8av9jw4cwf/F5HVRQ=",
         "KzsrnOhCq4tqbGFMsflgS7ig1QLRr0nFJrcrEIlOlbU="
-    ] 
+    ]
 }
 ```
-
-Here's an encrypted identity mapping request format with placeholder values:
-
-```sh
-echo '[Unencrypted-JSON-Request-Body]' \
-  | encrypt_request.py [CLIENT_SECRET] \
-  | curl -X POST 'https://prod.euid.eu/v2/identity/map' -H 'Authorization: Bearer [CLIENT_API_KEY]' -d @- \
-  | decrypt_response.py [CLIENT_SECRET] 
+```json
+{
+    "phone":[
+        "+12345678901",
+        "+441234567890"
+    ]
+}
+```
+```json
+{
+    "phone_hash":[
+        "EObwtHBUqDNZR33LNSMdtt5cafsYFuGmuY4ZLenlue4=",
+        "Rx8SW4ZyKqbPypXmswDNuq0SPxStFXBTG/yvPns/2NQ="
+    ]
+}
 ```
 
-Here's an encrypted identity mapping request example for email addresses:
+Here's an encrypted identity mapping request example for a phone number:
 
 ```sh
-echo '{"email": ["user@example.com", "user2@example.com"]}' \
-  | encrypt_request.py DELPabG/hsJsZk4Xm9Xr10Wb8qoKarg4ochUdY9e+Ow= \
-  | curl -X POST 'https://prod.euid.eu/v2/identity/map' -H 'Authorization: Bearer YourTokenBV3tua4BXNw+HVUFpxLlGy8nWN6mtgMlIk=' -d @- \
-  | decrypt_response.py DELPabG/hsJsZk4Xm9Xr10Wb8qoKarg4ochUdY9e+Ow= 
+echo '{"phone": ["+12345678901", "+441234567890"]}' | python3 uid2_request.py https://prod.euid.eu/v2/identity/map YourTokenBV3tua4BXNw+HVUFpxLlGy8nWN6mtgMlIk= DELPabG/hsJsZk4Xm9Xr10Wb8qoKarg4ochUdY9e+Ow=
```

Comment:
this is ported correctly correctly from UID2 but i think we should improve here and UID2 to easily indicate which param is what and we can just replicate what's on the token/generate page edits and use:
 
```python3 uid2_request.py [Your-Client-API-Key] [Your-Client-Secret] ```

Reply:
@sunnywu Done, thx. 
For the record not `python3 uid2_request.py [Your-Client-API-Key] [Your-Client-Secret]`  but `python3 uid2_request.py https://prod.euid.eu/v2/identity/map [Your-Client-API-Key] [Your-Client-Secret]`. Hope that's correct.